Core/Core/Configuration/Config/UIComponentsConfig.swift

```diff
@@ -11,16 +11,19 @@ private enum Keys: String {
     case courseNestedListEnabled = "COURSE_NESTED_LIST_ENABLED"
     case courseTopTabBarEnabled = "COURSE_TOP_TAB_BAR_ENABLED"
     case courseBannerEnabled = "COURSE_BANNER_ENABLED"
+    case courseUnitProgressEnabled = "COURSE_UNIT_PROGRESS_ENABLED"
 }
 
 public class UIComponentsConfig: NSObject {
-    public var courseNestedListEnabled: Bool = false
+    public var courseNestedListEnabled: Bool
     public var courseBannerEnabled: Bool
+    public var courseUnitProgressEnabled: Bool
     public var courseTopTabBarEnabled: Bool
 
     init(dictionary: [String: Any]) {
         courseNestedListEnabled = dictionary[Keys.courseNestedListEnabled.rawValue] as? Bool ?? false
         courseBannerEnabled = dictionary[Keys.courseBannerEnabled.rawValue] as? Bool ?? false
+        courseUnitProgressEnabled = dictionary[Keys.courseUnitProgressEnabled.rawValue] as? Bool ?? false
```

Comment:
You can inherit the Keys enum from `RawStringExtractable` so that you don't have to mention `.rawValue` while extracting the data from dictionary.